cert/cert.go

```diff
@@ -45,6 +45,10 @@ const (
 	duration365d = time.Hour * 24 * 365
 )
 
+var (
+	ErrStaticCert = errors.New("cannot renew static certificate")
```

Comment:
Is this value used outside of this package?

Reply:
it is used outside dynamiclistener/cert, in dynamiclistener/factory and dynamiclistener, but not outside this module.